src/main/java/net/openhft/chronicle/queue/reader/MessageConsumer.java

```diff
@@ -0,0 +1,18 @@
+package net.openhft.chronicle.queue.reader;
+
+/**
+ * Message consumers make a chain-of-responsibility pattern, they
```

Comment:
They don't yet, but they could you could do includeFilter -> excludeFilter -> index Prepender -> sink

I just left it as it was to minimise the change